src/javascripts/modules/pills.js

```diff
@@ -1,24 +1,55 @@
 export default class Pills {
   constructor(el) {
     this.el = el
+    this.setVars()
     this.setupListeners()
   }
 
+  setVars() {
+    this.ticking = false
+    this.lastScrollY = 0
+  }
+
   setupListeners() {
     window.addEventListener('load', () => this.handleLoad())
+    window.addEventListener('scroll', () => this.handleScroll())
   }
 
   handleLoad() {
     this.timeout = setTimeout(() => this.animatePills(), 500)
     this.releaseSqueezeTimeout = setTimeout(() => this.releasePillSqueeze(), 3500)
   }
 
+  handleScroll() {
+    this.lastScrollY = window.scrollY
+    if (!this.ticking) {
+      window.requestAnimationFrame(() => {
+        this.shakePills()
```

Comment:
Maybe call this `shiftPills()` (instead of shake) since the style of the animation is different now, or something more generic, like `transitionPills()` or `movePills()`.